doc/man1/req.pod

```diff
@@ -255,6 +256,14 @@ be a positive integer. The default is 30 days.
 Serial number to use when outputting a self signed certificate. This
 may be specified as a decimal value or a hex value if preceded by B<0x>.
 
+=item B<-addext ext>
+
+Add a specific extension to the certificate (if the B<-x509> option is
+present) or certificate request.  The argument must have the form of
+a key=value pair as it would appear in a config file.
+
+This option can be given multiple times, its values will be accumulated.
```

Comment:
But if it's "its", then it ought to be "its value", i.e. singular...

Reply:
Mm, I'm unsure about that...  It's *one* option, but when given multiple times, there are more than one *value*...  Err, @mattcaswell, @t-j-h, @richsalz, you guys are much more native speakers than @dot-asm or me, care to make a final verdict?

Reply:
Actually I'm not sure what is technically correct here, but I would write it the way it is currently written.

Reply:
I would say "the values are accumulated"

Reply:
Plural. The "its" binds to the whole sentence before the comma (my recollection of the rules). 
So I would have written the sentence the same was as @levitte - although you could avoid the issue entirely with "each value is ..." and I would also avoid "accumulated" as that is not what is happening - accumulated has no implied ordering - it is a pure collection - and these are actually "added" or "appended" ... and I would tend towards "added" given the API usage.